src/structures/Guild.js

```diff
@@ -518,6 +525,7 @@ class Guild extends Base {
    * @readonly
    */
   get nameAcronym() {
+    if (!this.name) return null;
```

Comment:
```suggestion
  get nameAcronym() {
```
out of the scope of this pr